forms-flow-web/src/components/FOI/customComponents/StateDropDown.js

```diff
@@ -94,37 +98,37 @@ const StateDropDown = ({
     if (_isMinistryCoordinator) {
       _stateList = MinistryStateList;
     }
-    const personalRequest = requestType?.toLowerCase() === FOI_COMPONENT_CONSTANTS.REQUEST_TYPE_PERSONAL;
+    const personalRequest =
+      requestType?.toLowerCase() ===
+      FOI_COMPONENT_CONSTANTS.REQUEST_TYPE_PERSONAL;
     const personalIAO = !_isMinistryCoordinator && personalRequest;
-
+    const previousState =
+      stateTransition?.length > 0 && stateTransition[1]?.status;
```

Comment:
declared previousState in the top to check whether it is feeestimate or response. Based on that the onhold state list will be returned.